src/test/java/edu/harvard/iq/dataverse/api/MetadataBlocksIT.java

```diff
@@ -2,25 +2,59 @@
 
 import com.jayway.restassured.RestAssured;
 import com.jayway.restassured.response.Response;
-import static javax.ws.rs.core.Response.Status.OK;
 import org.hamcrest.CoreMatchers;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static javax.ws.rs.core.Response.Status.CREATED;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
 public class MetadataBlocksIT {
 
-    @BeforeClass
+    @BeforeAll
     public static void setUpClass() {
         RestAssured.baseURI = UtilIT.getRestAssuredBaseUri();
     }
 
     @Test
-    public void testGetCitationBlock() {
+    void testGetCitationBlock() {
         Response getCitationBlock = UtilIT.getMetadataBlock("citation");
         getCitationBlock.prettyPrint();
         getCitationBlock.then().assertThat()
                 .statusCode(OK.getStatusCode())
                 .body("data.fields.subject.controlledVocabularyValues[0]", CoreMatchers.is("Agricultural Sciences"));
     }
+    
+    @Test
+    void testDatasetWithAllDefaultMetadata() {
+        // given
+        Response createUser = UtilIT.createRandomUser();
+        assumeTrue(createUser.statusCode() < 300,
+            "code=" + createUser.statusCode() +
+            ", response=" + createUser.prettyPrint());
```

Comment:
This is pretty different than our usual style but whatever. 😄

Reply:
I'm trying to keep to good practice. Creating the user etc is not the test, it's a requirement. So the test shall be aborted if requirements ain't met. Using an assertion for this is not the same, as it claims to make this part to be in scope for testing. 

Also, providing more useful error messages here by also printing the actual status code helps if things go sideways. Most other tests are just creating a lot of noise by always doing the pretty printing thing.

Reply:
Let's talk about it some day.